pages/custom-query-modules/manage-query-modules.mdx

```diff
@@ -124,7 +124,14 @@ update was successful.
 
 ### `mg.load`
 
-Loads or reloads the given module. A module can only be reloaded if it is not in use. If the module that would be reloaded is being used by another thread while this is run then this will fail the error: `Unable to unload module 'module_name', it is currently being used`. To resolve this rerun the procedure when the module isn't in use.
+Loads or reloads the given module. Module name provided in the input file is a string of the module name, without the file extension.
+A module can only be reloaded if it is not in use, and if it exists in the module directory.
+If the module that would be reloaded is being used by another thread while this is run then this will fail
```

Comment:
```suggestion
Loads or reloads the given module. The module name provided in the input file is a module name string without the file extension.
A module can only be reloaded if it is not in use and if it exists in the module directory.
If the module that would be reloaded is being used by another thread while this is run, then this will fail
```